_includes/resource-item.html

```diff
@@ -3,9 +3,12 @@
     <h2 class="u-hiddenVisually">
       {{ include.title }}
     </h2>
-    {% if include.image %}
-    <img class="Card-mainObject" alt=""
-      src="{{ include.image | prepend: site.url }}">
+    {% if include.image and include.imageExt %}
+    <img
+      class="Card-mainObject"
+      alt=""
+      src="{{ include.image }}/2x.{{ include.imageExt }}"
```

Comment:
If there are no longer two images, are the separate attributes for extension and image still necessary?

Reply:
Should I change it to just:

```yml
image: /images/foo/2x.png
```

Reply:
I'm not sure, I don't 100% know how the changes are working to be honest.

I think the main thing I'm critiquing is that the icons were already something of a pain point to maintain, because they require a modestly verbose bit of frontmatter (adding `images/` every time) and you need to spelunk through manifests or look for touch icons in the first place.

So what I'm slightly sensitive to is that we are now having to remember another segment (`/2x`) and potentially a separate frontmatter attribute (`imageExt`). It seems like maybe adding complexity to something that, relative to the rest of the contribution process, is already kinda complex.

But I might be wrong. Other contributors like @grigs, @gerardo-rodriguez and @derekshirk might have different opinions. Maybe since it's already pretty manual, making it _more_ manual isn't a big deal? ¯\\\_(ツ)_/¯

Reply:
I started on a script to fetch a PWA icon and put it into the `images/` folder. That will help reduce the manual work. I'll make a PR once this PR is merged.

Is there value in keeping the original-quality image as well as the downsized/optimized one? My initial thought there was that maybe in the future we would decide to use a larger size than 122x122. If we decided to switch to just having a single image (not keeping the originals), it would make things a lot easier on the frontmatter side of things. What do you think?

Reply:
I'm trying to track down what problem we're trying to solve here. Can someone point me to what prompted the changes?

Reply:
The main question right now is this:

For the PWA icons, do we want to:

1. Make the image resizer/optimizer overwrite the images
2. Keep copies of the original (larger) images, and save the smaller optimized images to separate files.

Reply:
As a general rule, I like to keep the highest quality asset available per https://cloudfour.com/thinks/8-guidelines-and-1-rule-for-responsive-images/#2-encourage-people-to-upload-the-highest-quality-source-possible

Reply:
OK, so now the question is:

If we are storing the images like:

```
images/
  asdf/
    original.png
    2x.png # resized/optimized
```

How do we specify this in the frontmatter? Should we do:

```yml
image: asdf
imageExt: png
```

or

```yml
image: asdf.png # even though there is no actual file named asdf.png
```

Reply:
@tylersticka and/or @grigs which of these options do you prefer, or do you have  another idea?

Reply:
Here's a crazy thought, @calebeby: Could we say "name the files the same as the slug," and then not bother with the image filename at all?

Maybe that's not possible since there's the file extension, just an idea that sparked when I read "there is no actual file named asdf.png".

Reply:
Good idea! We could do that. Then we could _just_ have the `imageExt` property (we could make it default to png).